docs/pages/common_issues/bundle_size.mdx

```diff
@@ -0,0 +1,39 @@
+import {Callout} from 'nextra/components'
+
+
+#### Reducing Bundle Size
+
+Next will incorrectly trace dev dependencies to be included in the output `node_modules`, which will significantly increase the lambda bundle. For example, the @swc/core-\* binary is ~33MB!
```

Comment:
With 13.5.1+ they probably will not even end up in `node_modules`, a big part is now bundled by next and may end up in one of the bundle

Reply:
I built w/ 13.5.4 and the `core-darwin-arm64` binary was still there unfortunately